src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs

```diff
@@ -25,7 +25,9 @@ public static class ServiceStackRedisIntegration
         [InterceptMethod(
             CallerAssembly = "ServiceStack.Redis",
             TargetAssembly = "ServiceStack.Redis",
-            TargetType = "ServiceStack.Redis.RedisNativeClient")]
+            TargetType = "ServiceStack.Redis.RedisNativeClient",
+            TargetMinimumVersion = Major4,
+            TargetMaximumVersion = Major4)]
```

Comment:
Maximum version should be "5". `ServiceStack.Redis` is up to `5.5.0` and afaik the integration works with 5.x.
https://www.nuget.org/packages/ServiceStack.Redis/